src/ModbusSlave.cpp

```diff
@@ -615,7 +694,7 @@ uint8_t Modbus::createResponse()
         _responseBufferLength += 1;
 
         // execute callback and return the status code
-        return Modbus::executeCallback(CB_READ_EXCEPTION_STATUS, 0, 8);
+        return Modbus::executeCallback(_requestBuffer[MODBUS_ADDRESS_INDEX], CB_READ_EXCEPTION_STATUS, 0, 8);
```

Comment:
can we extract `_requestBuffer[MODBUS_ADDRESS_INDEX]` to a helper variable ?

for example:
```
uint16_t requestUnitAddress = _requestBuffer[MODBUS_ADDRESS_INDEX];
```
...
```
return Modbus::executeCallback(requestUnitAddress, CB_READ_EXCEPTION_STATUS, 0, 8);
```